.storybook/Heading.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Heading } from '../src'
+import { fontSizes, colors } from '../src/theme'
+
+const description = 'A type of the <Text> component' +
+  ' using HTML h1-h6 element for setting section headings,' +
+  ' extending all <Text> properties'
+
+storiesOf('Heading', module)
+  .add('Heading component', withInfo({
+    inline: true,
+    text: description
+  })(() => (
+    <Heading m={3}>Heading component</Heading>
+  )))
+  .add('Using dot-notation with h1-h6', () => (
+    <section>
+      <Heading.h1>Heading h1</Heading.h1>
+      <Heading.h2>Heading h2</Heading.h2>
+      <Heading.h3>Heading h3</Heading.h3>
+      <Heading.h4>Heading h4</Heading.h4>
+      <Heading.h5>Heading h5</Heading.h5>
+      <Heading.h6>Heading h6</Heading.h6>
+    </section>
+  ))
+  .add('Using <Text> properties', () => (
+    <section>
+      <Heading align='left' fontSize={fontSizes[6]} color={colors['green']}>
```

Comment:
Any reason why you use fontSize instead of <Heading.h1> in this story?

Reply:
For heading elements, it can be useful to change the font size, but keep the semantically correct markup, e.g. an `<h2>` that looks like an `<h1>` – I'm guessing that's what this story is demonstrating here.

That said, the theme values shouldn't need to be imported in this module and can be rewritten as `fontSize={6} color='green'`, which will pick up values from the theme context

Reply:
Great feedback. Agree on the same thought that the `fontSize` prop would be semantically more expressive. If users opt out not to use `fontSize` prop, the `Heading` component defaults the font sizes with reasonable values from the `space` variable from theme.

Updated the `Heading` story to use direct number and string input, instead of overhead with `fontSizes` and `colors` variable. +1 on this feedback.

Thanks~ 👍